### ISSUE TITLE
Data augmentation strategy includes translation of odd/even slices  

### ISSUE THREAD

**Issue by naga-karthik** [ivadomed/canproco#19]
Some 2D scans (e.g. PSIR, STIR) include images that are acquired by combining odd and even slices separately. This is susceptible to patient motion as consecutive slices can have a larger duration to be acquired. This also leads to challenging issues with registration to 3D scans (e.g T2w). 

Suggestion by @jcohenadad: A good data augmentation strategy would be to shift/translate odd and even slices in a scan for training training a multi-contrast model.

**Comment by jcohenadad** [comment]
> Some 2D scans (e.g. PSIR, STIR) include images that are acquired by combining odd and even slices separately.

Not exactly. This is called a 'sequential' 2D acquisition wherein _each_ slice is acquired separately, and the timing is 2, 4, 6, ... 1, 3, 5, etc. So "odd" and "even" are _not_ acquired as two separate blocks, but rather do manifest as two separate entities in that their timing is shifted. Eg: it is possible that motion would occur at the last 10% of the acquisition, in which case odd/even would be perfectly aligned from slices 1->n, but then screwed up from n+1->m.

**Comment by naga-karthik** [comment]
The section "[Multislice imaging](https://www.cis.rit.edu/htbooks/mri/chap-8/chap-8.htm#8.2)" in this chapter gives a good explanation/more details of how the acquisition is done.
